st2api/st2api/controllers/v1/auth.py

```diff
@@ -102,9 +102,18 @@ def get_all(self, requester_user, show_secrets=None, limit=None, offset=0):
         mask_secrets = self._get_mask_secrets(show_secrets=show_secrets,
                                               requester_user=requester_user)
 
-        if limit and int(limit) > self.max_limit:
-            msg = 'Limit "%s" specified, maximum value is "%s"' % (limit, self.max_limit)
-            raise ValueError(msg)
+        if limit:
```

Comment:
It looks like this code is duplicated in two places and could perhaps be refactored in a utility function.

Reply:
That's my only "blocker", besides that, LGTM.

I would also like to reach some kind of consensus on that - https://github.com/StackStorm/st2/pull/3761#issuecomment-356313645